docs/source/starter/introduction_guide.rst

```diff
@@ -127,9 +128,45 @@ Now we add the training_step which has all our training loop logic
             loss = F.nll_loss(logits, y)
             return loss
 
+Optimizer
+---------
+
+Next we choose what optimizer to use for training our system.
+In PyTorch we do it as follows:
+
+.. code-block:: python
+
+    from torch.optim import Adam
+
+    optimizer = Adam(LitMNIST().parameters(), lr=1e-3)
+
+
+In Lightning we do the same but organize it under the :func:`~pytorch_lightning.core.LightningModule.configure_optimizers` method.
+
+.. testcode::
+
+    class LitMNIST(LightningModule):
+        def configure_optimizers(self):
+            return Adam(self.parameters(), lr=1e-3)
+
+.. note:: The LightningModule itself has the parameters, so pass in self.parameters()
+
+However, if you have multiple optimizers use the matching parameters
+
+.. testcode::
+
+    class LitMNIST(LightningModule):
+        def configure_optimizers(self):
+            return Adam(self.generator(), lr=1e-3), Adam(self.discriminator(), lr=1e-3)
+
 Data
 ----
 
+Install torchvision library.
+
+.. code-block:: shell
+
+    pip install torchvision
```

Comment:
Reviewers: Placing this installation guide here could be distracting to some users maybe? Instead, should there be an installation guide for all extra packages at the very top of the page?

e.g. 

```
Install extra packages used in this tutorial:

.. code-block:: shell

    pip install torchvision transformers ...

```

Reply:
Agree this may be distracting and breaks the flow. Pretty sure this can be left out entirely. If the user does not know how to install this Python package while seeing an import error, they are probably not ready to use Lightning at this point anyway.

Reply:
> Pretty sure this can be left out entirely.

That's what I thought, too. If we do this, we should do this to the entire docs for consistency, which can be quite redundant and less readable...

@os1ma Could you revert the change here at least for now?

Reply:
Thanks for the review!
I agree that this `pip install` explanation is not necessary for Lightning users. I will remove this part.